modules/iyzicocheckoutform/iyzicocheckoutform.php

```diff
@@ -148,8 +142,97 @@ protected function _postProcess()
         $this->_html .= $this->displayConfirmation($this->l('Settings updated'));
     }
 
-    protected function _displayIyzicoInfo()
-    {
+    protected function _displayIyzicoInfo() {
+        $version = $this->_ModuleVersion;
+        $psver = _PS_VERSION_;
+        $ch = curl_init();
```

Comment:
Bu curl isteği kapatılmamış. Fazladan kaynak kullanılmaması için curl responsu alındıktan (154.satır) sonra kapatmak gerek.
`curl_close($ch)`